.github/workflows/pr_basic_compilation_check.yml

```diff
@@ -1,7 +1,7 @@
 name: Compile check
 
 # Controls when the action will run. Triggers the workflow on push or pull request
```

Comment:
Please delete `push or`

Reply:
Next commit will remove this line because this comment is meaningless.